src/instructlab/model/model.py

```diff
@@ -0,0 +1,52 @@
+# from instructlab.lab import ilab
+# Standard
+import multiprocessing
+
+# Third Party
+from click_didyoumean import DYMGroup
+import click
+
+# First Party
+from instructlab import helpers
+
+# Local
+from .chat import chat
+from .convert import convert
+from .download import download
+from .evaluate import evaluate
+from .list import list
+from .serve import serve
+from .train import train
+
+multiprocessing.set_start_method(
+    helpers.DEFAULT_MULTIPROCESSING_START_METHOD, force=True
+)
+
+
+@click.group(cls=DYMGroup, chain=True)
+@click.version_option(package_name="instructlab")
+@click.option(
+    "--config",
+    "config_file",
+    type=click.Path(),
+    default=helpers.DEFAULT_CONFIG,
+    show_default=True,
+    help="Path to a configuration file.",
+)
+@click.pass_context
+# pylint: disable=redefined-outer-name
+def model(ctx, config_file):
+    """Command Group for Interacting with the Models in InstructLab.
+
+    If this is your first time running ilab, it's best to start with `ilab init` to create the environment.
```

Comment:
Broken record here but think we might want this to be `ilab config init`

Reply:
https://github.com/instructlab/instructlab/issues/1331